.scripts/publish.yaml

```diff
@@ -6,28 +6,41 @@
 trigger:
 - v2 
 
-pool:
-  vmImage: 'ubuntu-latest'
-
-steps:
-- task: NodeTool@0
-  inputs:
-    versionSpec: '10.x'
-  displayName: 'Install Node.js'
-
-- script: |
-    npm install -g publish-release@1.6.1
-
-    # npm install 
-    npm install
-    rc=$?; if [ $rc -ne 0 ]; then exit $rc ; fi 
-
-    # build packages 
-    npm run publish-preview
-    
-    # publish autorest core to a github release.
-    v=`dir src/autorest-core/*.tgz  | sed 's/.*\(2\.0\..*\)\.tgz/\1/g'`
-    publish-release --token $(azuresdk-github-pat) --repo autorest --owner azure --name autorest-core-$v --tag autorest-core-$v --notes='v2 prerelease build' --prerelease --editRelease false --assets ./src/autorest-core/microsoft.azure-autorest-core-$v.tgz --target_commitish $(Build.SourceBranchName)
-    rc=$?; if [ $rc -ne 0 ]; then exit $rc ; fi 
-    cd ..
+extends:
+  template: /eng/pipelines/templates/1es-redirect.yml
+  parameters:
+    stages:
+    - stage: Publish_Stage
+      variables:
+        - template: /eng/pipelines/templates/variables/image.yml
```

Comment:
If you're using the stage/variable expansion model from Patrick, then wouldn't we also want to move the image.yml inclusion inline to 1es-redirect?

Reply:
/shrug. it works.